include/akushon/config/utils/config.hpp

```diff
@@ -23,6 +23,7 @@
 
 #include <fstream>
 #include <string>
+#include "nlohmann/json.hpp"
```

Comment:
Should use angled brackets for external libraries:
```suggestion
#include <nlohmann/json.hpp>
```